src/serializers/resource.js

```diff
@@ -42,7 +42,31 @@ function ResourceSerializer(Implementation, model, records, integrator, opts, me
   this.perform = function () {
     var typeForAttributes = {};
 
-    function getAttributesFor(dest, fields) {
+    function getRelatedLinkForHasMany(modelName, relationshipName) {
+      return function (record, current, parent) {
+        if (current && current[relationshipName]) {
```

Comment:
We do not want the link to be set if the relation is already included in the payload